mmdet/apis/inference.py

```diff
@@ -175,13 +175,21 @@ def show_result_pyplot(model,
         score_thr (float): The threshold to visualize the bboxes and masks.
         fig_size (tuple): Figure size of the pyplot figure.
         title (str): Title of the pyplot figure.
-        block (bool): Whether to block GUI.
+        block (bool): Whether to block GUI. Default: True
+        wait_time (float): Value of waitKey param.
+                Default: 1.
     """
     if hasattr(model, 'module'):
         model = model.module
-    img = model.show_result(img, result, score_thr=score_thr, show=False)
-    plt.figure(figsize=fig_size)
-    plt.imshow(mmcv.bgr2rgb(img))
-    plt.title(title)
-    plt.tight_layout()
-    plt.show(block=block)
+    if block:
+        wait_time = 0
```

Comment:
Either deprecate `block` with warning or assert the consistency between `block` and `wait_time`.